templates/zerver/help/deactivate-or-reactivate-a-user.md

```diff
@@ -28,6 +28,33 @@ user's bots will also be deactivated. Lastly, the user will be unable to
 create a new Zulip account in your organization using their deactivated
 email address.
 
+### Notify users of their deactivation
+
+If you want to notify the user of their deactivation,
+you can do so by sending them a email.
+
+{start_tabs}
+
+{settings_tab|user-list-admin}
+
+ 2. Click the **Deactivate** button to the right of the user account that you
+want to deactivate.
+
+ 3. Check the checkbox labeled **"Send a notification email to \[user]?"**.
```

Comment:
Reading this, I'm reminded that we generally aim to not put details like the user's name in the checkbox.  Maybe call the checkbox "Notify this user by email" both here and in the actual UI?